package.json

```diff
@@ -22,6 +22,7 @@
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
     "express": "^4.18.1",
+    "fs": "^0.0.1-security",
```

Comment:
fs is something available to us through node. We don't have to install it. 

I'm not sure what this package is but I don't think it what you expect :)